### ISSUE TITLE
Get-HPOVLdapDirectory -- the property 'sessionId' cannot be found on this object

### ISSUE THREAD

**Issue by erikgraa** [HewlettPackard/POSH-HPOneView#159]
Please fill in as much information as possible to help resolve your issue.

### Actual Behavior
```
Get-HPOVLdapDirectory: The property 'sessionId' cannot be found on this object. Verify that the property exists and can be set.
```

### Steps to reproduce
Connect-HPOVMgmt
Get-HPOVLdapDirectory

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion`): 2.0.530.1
HPE OneView Appliance Version: 2.00.07

**Comment by ChrisLynchHPE** [comment]
I do not understand how that Cmdlet regressed.  There was no work performed on that Cmdlet, nor `Send-HPOVRequest`, which is where the error was originating.  I've added what should be a fix, but feels more like a workaround.  I've tested that it does not affect other Cmdlets.  So, I'll get this added to the next milestone release.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 2.0.576.0](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v2.0.576.0).

Closing.
